ui/src/utils/visualisations/aggregationGroup/grouper/index.spec.js

```diff
@@ -1,13 +1,79 @@
 import grouper from './index';
 
-test('aggrigationGroup grouper adds unwind pipeline stage', () => {
-  const result = grouper({
-    valueType: 'statements',
-    groupType: 'statement.context.contextActivities.category',
-    operatorType: 'uniqueCount'
+describe('aggregationGroup grouper', () => {
+  describe('when groupType is "date"', () => {
+    const result = grouper({
+      valueType: 'statements',
+      groupType: 'date',
+      operatorType: 'uniqueCount'
+    });
+
+    test('pipeline has correct stages', () => {
+      expect(result.toJS().map(stage => Object.keys(stage)[0]))
+        .toEqual([
+          '$project',
+          '$group',
+          '$sort',
+          '$limit',
+          '$project'
+        ]);
+    });
   });
 
-  expect(result.getIn([1, '$unwind']).toJS())
-    .toEqual({ path: '$statement.context.contextActivities.category' });
-});
+  describe('when groupType starts with "statement.context.contextActivities."', () => {
+    const result = grouper({
+      valueType: 'statements',
+      groupType: 'statement.context.contextActivities.category',
+      operatorType: 'uniqueCount'
+    });
+
+    test('pipeline has correct unwind stage', () => {
+      expect(result.getIn([1, '$unwind']).toJS())
+        .toEqual({ path: '$statement.context.contextActivities.category' });
+    });
+  });
+
+  describe('when groupType starts with "persona.import."', () => {
+    const result = grouper({
+      valueType: 'statements',
+      groupType: 'persona.import.abcde',
+      operatorType: 'uniqueCount'
+    });
+
+    test('pipeline has correct stages', () => {
+      expect(result.toJS().map(stage => Object.keys(stage)[0]))
+        .toEqual([
+          '$lookup',
+          '$unwind',
```

Comment:
Required if implementing my above suggestion

```suggestion
          '$match',
          '$unwind',
```